packages/react-core/src/components/DataList/examples/DataList.md

```diff
@@ -86,14 +86,15 @@ import global_BorderWidth_sm from '@patternfly/react-tokens/dist/esm/global_Bord
 
 ### Draggable
 
-Draggable data lists used to have their own HTML5-based API for drag and drop, which wasn't able to fulfill requirements such as custom styling on items being dragged. So we wrote generic `DragDrop`, `Draggable`, and `Droppable` components for this purpose. Use those new components instead of the deprecated (and buggy!) HTML5-based API.
+Note: There is a new recommended drag and drop implementation that has full keyboard functionality. Examples and details may be located [here](/components/drag-and-drop/react-next).
 
-Note: Keyboard accessibility and screen reader accessibility for the `DragDrop` component are still in development.
+Draggable data lists used to have their own HTML5-based API for drag and drop, which wasn't able to fulfill requirements such as custom styling on items being dragged. So we wrote generic `DragDrop`, `Draggable`, and `Droppable` components for this purpose. Use those new components instead of the deprecated (and buggy!) HTML5-based API. Keyboard accessibility and screen reader accessibility for the `DragDrop` component are still in development.
```

Comment:
```suggestion
Previously, draggable data lists had their own API for the [drag and drop component](/components/drag-and-drop), which wasn't able to fulfill requirements like custom styling for items as they are dragged. 

To address this disparity, `<DragDrop>`, `<Draggable>`, and `<Droppable>` components were added to replace our now deprecated HTML5-based API. 

Note that keyboard and screen reader accessibility functionality for the `<DragDrop>` component are still in development.
```

I'm inclined to reword the "fulfill requirements" part of the first sentence, because I'm not quite understanding what is meant by that, but I can't tell what's important to convey. What kind of requirements is this referring to - accessibility? Were there others besides "custom styling" that could help buff out this sentence? The way it reads now feels incomplete, but maybe it's just because I'm unfamiliar with the previous implementation

Reply:
I think it's referring to general flexibility of the implementation, and "requirements" in the sense of what a user might want or need. 

`Previously, draggable data lists had their own API for the [drag and drop component](/components/drag-and-drop), which wasn't flexible enough to allow custom styling for items as they are dragged.` Does that sound better?

Reply:
Yeah that sounds awesome! Thanks for clarifying